jjb/dynamic/scale-ci_cluster-density.yml

```diff
@@ -0,0 +1,168 @@
+- job:
```

Comment:
We need to add a build script which reads the properties files and calls the cluster density job when selected to run as part of the pipeline. For example: https://github.com/openshift-scale/scale-ci-pipeline/pull/61/files.

Reply:
yup, WIP